core/src/banking_stage.rs

```diff
@@ -2266,10 +2266,9 @@ mod tests {
                 Blockstore::open(ledger_path.path())
                     .expect("Expected to be able to open database ledger"),
             );
-            let poh_config = PohConfig {
-                target_tick_count: Some(bank.max_tick_height() + num_extra_ticks),
-                ..PohConfig::default()
-            };
+            let mut poh_config = PohConfig::default();
```

Comment:
Maybe overlooking something, but why this change?

Reply:
When one of the member variables is non-pub, the ".." method of filling in values no longer works. So instead, set it to default, then override the value that you want to be different.